pom.xml

```diff
@@ -20,8 +20,8 @@
       <alfresco.max.version>6.99.99</alfresco.max.version>
       <alfresco.google.config.version>1.0.0</alfresco.google.config.version>
       <alfresco.google.config.classifier>-community</alfresco.google.config.classifier>
-      <compiler-plugin.version>3.1</compiler-plugin.version>
-      <java.version>1.8</java.version>
+      <compiler-plugin.version>3.8.0</compiler-plugin.version>
+      <java.version>11</java.version>
```

Comment:
This should remain as 1.8 (to ensure 1.8 compatible byte code).

Reply:
Ok, I will revert it.

Reply:
Done. Thanks.